src/index.css

```diff
@@ -127,25 +124,26 @@
 .o-UsageBar__compact__bar {
   max-width: 500px;
```

Comment:
Wondering if we can remove this since it also imposes a style preference from within the child component.

Reply:
Hi @clarkmcc, this issue is now solved in the [1.2.1 version](https://github.com/ChrisUser/react-usage-bar/releases/tag/v1.2.1) of this project.
I've tried to remove all the strong style preferences hardcoded in the component.

Thanks for your feedback!

Reply:
This is awesome, thank you!